com.writerduet.writersolo.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: com.writerduet.writersolo
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: runwritersolo
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop
```

Comment:
This one is wrong.

Reply:
I don't even remember why I added that, I'll remove it.

Reply:
libappindicator usually needs `org.kde.StatusNotifierWatcher` and/or `com.canonical.indicator.application`. You want to check on KDE/GNOME (using extension) or other DEs that support tray icons.

Reply:
Which one in particular is wrong? The XCURSOR_PATH one?

Reply:
`--talk-name=org.freedesktop`

Reply:
I'm going to be a bit more lenient on the filesystem and just allow $HOME directory by default. I'll fix that talk as well. Thanks for bringing it to my attention.

Reply:
As for the KDE and GNOME notification ones, I'll add them right now.